### PR TITLE
[#PE-286][#PE-294] Changed storage to postgres and updated apim specs

### PR DESCRIPTION
**Pull request by michaeldisaro**

### CODE REVIEW

File: code/env/prod/terraform.tfvars [hunk 1]
```diff
@@ -30,24 +30,25 @@ cidr_subnet_redis         = ["10.0.8.0/24"]
...
-external_domain          = "pagopa.it"
-io_apim_resourcegroup    = "io-p-rg-internal"
-io_apim_name             = "io-p-apim-api"
-io_apim_productid        = "/subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b/resourceGroups/io-p-rg-internal/providers/Microsoft.ApiManagement/service/io-p-apim-api/products/cgnmerchant"
-io_apim_subscription_id  = "ec285037-c673-4f58-b594-d7c480da4e8b"
-redis_cache_family       = "C"
-redis_cache_sku_name     = "Standard"
-operator_search_capacity = 1
+database_name             = "cgnonboardingportal"
+activations_database_name = "cgnonboardingportal-activations"
```
Comment (AleDore) [inline code/env/prod/terraform.tfvars]: why not attribute-authority?

Comment (michaeldisaro) [reply code/env/prod/terraform.tfvars]: Right! It's better.

Comment (michaeldisaro) [reply code/env/prod/terraform.tfvars]: fixed in https://github.com/pagopa/cgn-onboarding-portal-infra/pull/82/commits/9d084f39c33fbfe0b2f6bbc7123b656e925b19ca

**Review by pasqualedevita — APPROVED** [review]

**Comment by pasqualedevita** [comment]
applied in uat

**Review by uolter — APPROVED** [review]

**Review by uolter — APPROVED** [review]

**Pull request merged**